src/index.js

```diff
@@ -0,0 +1,7 @@
+import view from './js/views/index';
```

Comment:
You can omit the `index` on the path as it is going to look for it by default if no file is given. Not an issue really and I'm sure you know that, but just in case 😄 : `import view from './js/views`